examples/simple-genai-server/README.md

```diff
@@ -0,0 +1,75 @@
+# Simple Game Server for GenAI
+
+A simple game server that serves as an example of how to integrate GenAI endpoints into your
+dedicated game server. You can either interact with one GenAI endpoint via TCP, or set up two
+endpoints to "chat" to each other.
+
+## Setting up the GenAI Inference Server
+
+You will need a separate GenAI inference server. This game server uses the
+[Google for Games GenAI](https://github.com/googleforgames/GenAI-quickstart) as its inference server.
+This particular inference server has the request structure for the /chat endpoint:
+
+```
+type GenAIRequest struct {
+	Context         string  `json:"context,omitempty"`
+	Prompt          string  `json:"prompt"`
+}
+```
+
+If you need a different request structure, you will need to clone or fork this repository and
+modify this in `main.go`. Update `REPOSITORY` in the `agones/examples/simple-genai-server/Makefile`
```

Comment:
Can you turn `main.go` and `agones/examples/simple-genai-server/Makefile` into links to the files?

Reply:
Oh yep, looks like GitHub can do relative links.